result_set.go

```diff
@@ -130,18 +151,18 @@ func genNode(vertex *nebula.Vertex) (*Node, error) {
 	}, nil
```

Comment:
need set timezone and add tests

Reply:
This senatio is tested in the `TestServiceDataIO` of `client_test.go` at line 257